EIPS/eip-3074.md

```diff
@@ -0,0 +1,179 @@
+---
+eip: 3074
+title: Sponsored Transaction Precompile
+author: Sam Wilson (@SamWilsn)
+discussions-to: <URL>
+status: Draft
+type: Standard
+category: Core
+created: 2020-10-15
+---
+
+## Simple Summary
+
+Creates a new precompile, analogous to `CALL`, that sets `msg.sender` based on an ECDSA signature.
+
+## Abstract
+
+We propose adding two precompiles:
+ - `CALL_PRECOMPILE` - forwards a `CALL`, setting `msg.sender` according to an ECDSA signature, using a relayer-sponsee nonce for replay protection.
```

Comment:
```suggestion
 - `CALL_PRECOMPILE` - forwards a `CALL`, setting `CALLER` according to an ECDSA signature, using a relayer-sponsee nonce for replay protection.
```

Reply:
Just a nit, but I generally prefer to not mix EVM and Solidity syntax. Feel free to disregard.